quickstarts/microsoft.documentdb/cosmosdb-private-endpoint/main.bicep

```diff
@@ -0,0 +1,85 @@
+@description('Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Virtual network name')
+param virtualNetworkName string
+
+@description('Cosmos DB account name (must be lower-case)')
+@maxLength(44)
```

Comment:
docs say this is 3-31 chars

Reply:
Portal:
![image](https://user-images.githubusercontent.com/6913354/122133192-1babfa00-cdf1-11eb-89ac-18adf3976ffb.png)

Reply:
fixed to match portql